docs/content/stable/reference/configuration/default-ports.md

```diff
@@ -79,11 +79,14 @@ YugabyteDB servers expose time series performance metrics in the [Prometheus exp
 <target>/prometheus-metrics
 ```
 
-You can access the Prometheus server on port `9090` of the YugabyteDB Anywhere node, and you can see the list of targets at the `http://<yugaware-ip>:9090/targets`. In particular, note port `9300` for node-level metrics:
+You can access the Prometheus server on port 9090 of the YugabyteDB Anywhere node, and you can see the list of targets at `http://<yugaware-ip>:9090/targets`. In particular, note port 9300 for node-level metrics:
 
-| Service      | Port |
-| ------------ | ---- |
-| Node metrics | 9300 |
+| Service           | Port |
+| ----------------- | ---- |
+| Prometheus server | 9090 |
+| Node metrics      | 9300 |
```

Comment:
We are conflating the prometheus port on YBA (9090) with the node exporter port (9300) on DB nodes. We should add 9300 to the list of ports needed open on the DB node.

We can work on this along with the network diagram request.